DistributionSystemApi/DistributionSystemApi.DataAccess/DataContext.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using DistributionSystemApi.Data.Interfaces;
+
+namespace DistributionSystemApi.Data
+{
+    public class DataContext : IDataContext
+    {
+        private readonly ContentDistributionSystemContext contentDistributionSystemContext;
+
+        public async void Create<TEntity>(TEntity entity, CancellationToken cancellationToken) where TEntity : BaseEntity
+        {
+            entity.Id = Guid.NewGuid();
+
+            contentDistributionSystemContext.Set<TEntity>().Add(entity);
+
+            await SaveChangesAsync(cancellationToken);
```

Comment:
we dont want to save changes there, we only want to wrap existed functionality of dbcontext